### PR TITLE
✨ Add Alpine container images

### PR DESCRIPTION
**Pull request by jacobwoffenden**
Add Alpine based images built with [apko](https://github.com/chainguard-dev/apko). Unable to use vanilla images as they run as root ☹️

This is directly related to #174, currently we cannot execute jobs on Cloud Platform.

### CODE REVIEW

File: .github/workflows/container-images.yml [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+
+      - name: Install cosign
+        id: install_cosign
+        uses: sigstore/cosign-installer@c3667d99424e7e6047999fb6246c0da843953c65 # v3.0.1
+
+      - name: Prepare build paramerters
+        id: prepare
+        run: |
+          echo "name=$(jq -r '.name' container-images/${{ matrix.images }}/config.json)" >> ${GITHUB_OUTPUT}
```
Comment (julialawrence) [inline .github/workflows/container-images.yml]: I think you can use github_env to pass values between steps. Github_outputs is for moving things between jobs.

File: .github/workflows/container-images.yml [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+            ghcr.io/ministryofjustice/data-platform-${{ steps.prepare.outputs.name }}:latest
+
+      - name: Push image
+        if: github.ref == 'refs/heads/main'
+        id: push_image
+        run: |
+          docker push ghcr.io/ministryofjustice/data-platform-${{ steps.prepare.outputs.name }}:${{ steps.prepare.outputs.version }}
+          docker push ghcr.io/ministryofjustice/data-platform-${{ steps.prepare.outputs.name }}:latest
+
+      - name: Sign image
```
Comment (julialawrence) [inline .github/workflows/container-images.yml:130]: The use of ghcr feels not so tactical at this point.

Comment (jacobwoffenden) [reply .github/workflows/container-images.yml:130]: 🐳 will move to DockerHub

**Review by julialawrence — APPROVED** [review]
LGTM

**Pull request merged**